src/js/text.js

```diff
@@ -0,0 +1,77 @@
+async function fetchNewMovies() {
+  const apiKey = 'd76d8df6a2b0d7e69c80b59c6dadfabe';
+  const response = await fetch(
+    `https://api.themoviedb.org/3/movie/now_playing?api_key=${apiKey}&language=en-US&page=1`
+  );
+  const data = await response.json();
+  return data.results;
+}
+
+function getRandomMovie(movies) {
+  const randomIndex = Math.floor(Math.random() * movies.length);
+  return movies[randomIndex];
+}
+
+function displayMovie(movie) {
+  document.getElementById(
+    'movie-image'
+  ).src = `https://image.tmdb.org/t/p/original/${movie.backdrop_path}`;
+  document.getElementById('movie-title').textContent = movie.title;
+  document.getElementById(
+    'movie-release-date'
+  ).textContent = `${movie.release_date}`;
+  document.getElementById('movie-rating').textContent = `${movie.vote_average}`;
+  document.getElementById('movie-genres').textContent = `${movie.genre_ids.join(
+    ', '
+  )}`;
+  document.getElementById('movie-description').textContent = movie.overview;
+  document.getElementById('voteRating').textContent = movie.vote_average;
+  document.getElementById('voteCount').textContent = movie.vote_count;
+}
+
+function updateLibraryButton(movie) {
+  const library = JSON.parse(localStorage.getItem('myLibrary')) || [];
+  const isInLibrary = library.some(item => item.id === movie.id);
+  const button = document.getElementById('library-button');
```

Comment:
button ul pe care il cauti are clasa, nu id. iar clasa este "uppcoming-button". verifica in html clasele sa fie corecte cu js ul